actions/targets.py

```diff
@@ -7,7 +7,7 @@ class PrometheusTarget(PrometheusAPI):
     def __init__(self, config):
         super(PrometheusTarget, self).__init__(config=config)
 
-    def run(self, url):
-        url_temp = self.url if url == "" else url
+    def run(self, url=None):
+        url_temp = self.url if url is None else url
```

Comment:
What if `url` specified in is an empty string `""` as before?
Is it worth checking for that too?

```py
url_temp = url if url else self.url
```
 ?

Reply:
Makes sense now, thanks for the explanation.